velox/functions/lib/aggregates/BitwiseAggregateBase.h

```diff
@@ -108,8 +108,7 @@ exec::AggregateRegistrationResult registerBitwise(
           case TypeKind::BIGINT:
             return std::make_unique<T<int64_t>>(resultType);
           default:
-            VELOX_CHECK(
-                false,
+            VELOX_FAIL(
```

Comment:
Does VELOX_UNREACHABLE fix the gcc warning? Thanks!

Reply:
Yes, VELOX_UNREACHABLE works too.  It makes sense to use that instead. I will update.

Reply:
Updated!